api/v1alpha1/baseboardmanagement_types.go

```diff
@@ -77,18 +85,19 @@ type BaseboardManagementStatus struct {
 	// +optional
 	Power PowerState `json:"powerState,omitempty"`
 
-	// BootDevice is the current first BootDevice of the BaseboardManagement.
+	// Conditions represents the latest available observations of an object's current state.
 	// +optional
-	// +kubebuilder:validation:Enum=Pxe;Disk;Bios;Cdrom;Safe
-	BootDevice BootDevice `json:"bootState,omitempty"`
+	Conditions []BaseboardManagementCondition `json:"conditions,omitempty"`
+}
 
-	// ErrorMessage represents cause of failure to set desired BaseboardManagement state.
+type BaseboardManagementCondition struct {
+	// Type of the BaseboardManagement condition.
 	// +optional
```

Comment:
I don't think this is optional? The message could be optional given the type can convey sufficient information.

Reply:
Ah good catch! Thanks 👍